backend/core/test/listings/listings.e2e-spec.ts

```diff
@@ -242,6 +245,124 @@ describe("Listings", () => {
     expect(modifiedListing.events[0].file.label).toBe(listingEvent.file.label)
   })
 
+  it("should return listings with AMI >= the filter value", async () => {
+    const paramsWithEqualAmi = {
+      view: "base",
+      limit: "all",
+      filter: {
+        $comparison: "NA",
+        ami: "60",
+      },
+    }
+    const res = await supertest(app.getHttpServer())
+      .get("/listings?" + qs.stringify(paramsWithEqualAmi))
+      .expect(200)
+    expect(res.body.items).toEqual(
+      expect.arrayContaining([
+        expect.objectContaining({ name: "Test: Default, Summary With 30 and 60 Ami Percentage" }),
+      ])
+    )
+
+    const paramsWithLessAmi = {
+      view: "base",
+      limit: "all",
+      filter: {
+        $comparison: "NA",
+        ami: "59",
+      },
+    }
+    const res2 = await supertest(app.getHttpServer())
+      .get("/listings?" + qs.stringify(paramsWithLessAmi))
+      .expect(200)
+    expect(res2.body.items).toEqual(
+      expect.arrayContaining([
+        expect.objectContaining({ name: "Test: Default, Summary With 30 and 60 Ami Percentage" }),
+      ])
+    )
+  })
+
+  it("should not return listings with AMI < the filter value", async () => {
+    const params = {
+      view: "base",
+      limit: "all",
+      filter: {
+        $comparison: "NA",
+        ami: "61",
+      },
+    }
+    const res = await supertest(app.getHttpServer())
+      .get("/listings?" + qs.stringify(params))
+      .expect(200)
+    expect(res.body.items).not.toEqual(
+      expect.arrayContaining([
+        expect.objectContaining({ name: "Test: Default, Summary With 30 and 60 Ami Percentage" }),
+      ])
+    )
+  })
+
+  it("should return listings with matching AMI in Units Summary, even if Listings.amiPercentageMax field does not match", async () => {
```

Comment:
I'm curious about another test case: one where a listing has two unit summaries, and one of them has a matching AMI percentage but the other doesn't have an AMI percentage at all. We expect the listing to be returned in that case, and I think it will be?

(I think this question of mine is a broader filtering question that's been answered elsewhere - essentially "if any one unitsummary within a listing matches the filtering criteria, will the listing be included in the results?". As I write this out, I believe that the answer is "yes" because the innerFilteredQuery will include that listing ID. So: no need to add another test case, but would you mind confirming my understanding, or letting me know what I've gotten wrong?

Reply:
This seed listing has one matching AMI unit and one not matching AMI unit summary, and the listing is shown (as we want). This is basicallythe same as haiving a listing with one matching unit and another unit without data, but I can add a test for that case if you'd like.

Reply:
No need to add a test case - I just wanted to confirm my understanding, thanks!